SYCL/Basic/kernel_bundle/kernel_bundle_api.cpp

```diff
@@ -1,8 +1,7 @@
-// Disable fallback assert here until online-support is fixed.
 // Use of per-kernel device code split and linking the bundle with all images
 // involved leads to multiple definition of AssertHappened structure due each
 // device image is statically linked against fallback libdevice.
-// RUN: %clangxx -DSYCL_DISABLE_FALLBACK_ASSERT=1 -fsycl -fsycl-device-code-split=per_kernel -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-device-code-split=per_kernel -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
I'm curious, how is this related to the bug that was fixed in intel/llvm#5476?

Reply:
This test started failing after I started check https://github.com/intel/llvm/pull/5476 . Moreover, there was a comment line ` //Disable fallback assert here until online-support is fixed.` I can guess, that it is about fixed bug.

Reply:
I resolved the problem. `Basic/kernel_bundle/kernel_bundle_api`  was affected indirectly. Before https://github.com/intel/llvm/pull/5476 it received a nullptr as a link option, but now here is passed an empty string.